tests/integration/targets/ios_acl_interfaces/tasks/cli.yaml

```
@@ -11,10 +11,19 @@
   set_fact: test_items="{{ test_cases.files | map(attribute='path') | list }}"
   delegate_to: localhost
 
-- name: Run test case (connection=ansible.netcommon.network_cli)
+- name: Run test case (connection=ansible.netcommon.network_cli with paramiko transport)
   include: '{{ test_case_to_run }}'
   vars:
     ansible_connection: ansible.netcommon.network_cli
```

```suggestion
    ansible_connection: ansible.netcommon.network_cli
    ansible_network_cli_ssh_type: paramiko
```

It's unnecessary, but would be more explicit about what we intend to test. And if we change the default down the line, continues to do what we intend